ext/drreg/drreg.c

```diff
@@ -186,12 +243,28 @@ find_free_slot(per_thread_t *pt)
     return MAX_SPILLS;
 }
 
+#ifdef X86
```

Comment:
These ifdefs are all going to have to be hunted down and changed when ARM support is added.  How will someone doing that distinguish them from permanently-x86 code such as for aflags?  I would make a new define SIMD_SUPPORTED or sthg local to this file, define it at the top for x86, and use it throughout.  That will make the code easier to read and make it easier to update for ARM.

Reply:
Done.